### ISSUE TITLE
Notes function does not refresh to show new note, nor clears the text box

### ISSUE THREAD

**Issue by rohinadalja** [dandahle/Catalyst-AppVetting#129]
The Notes function saves what's typed, but the page doesn't refresh to show the new note and clear the text entry box. I discovered this by adding notes to the project and the application vetting page.

=> Actual functionality is broken (success response clears it)… instead the POST just stalls.

**Comment by dandahle** [comment]
Bug fixes merged into update-setup branch work. Once this is merged into the production branch and pulled into the production AWS instance we will clear this issue as Done.
